include/seqan3/alignment/pairwise/policy/unbanded_dp_matrix_policy.hpp

```diff
@@ -66,22 +68,32 @@ class unbanded_dp_matrix_policy
         dimension_first_batch = std::ranges::size(first_batch);
         dimension_second_batch = std::ranges::size(second_batch);
 
+        current_column_index = 0;
+
         // We use only one column to compute the score.
         score_matrix.resize(dimension_second_batch + 1);
     }
 
     //!\brief Returns the current column of the alignment matrix.
-    auto current_column()
+    auto current_column() noexcept
     {
         using iter_t = std::ranges::iterator_t<decltype(score_matrix)>;
         return view::subrange<iter_t, iter_t>{std::ranges::begin(score_matrix), std::ranges::end(score_matrix)};
     }
 
+    //!\brief Moves internal matrix pointer to the next column.
+    constexpr void next_column() noexcept
+    {
+        ++current_column_index;
+    }
+
```

Comment:
this function is constexpr but the previous isn't. Is any of this supposed to run at compile-time? In any case it should be consistent, I think!

Reply:
and yes if constexpr allocator come into the standard, it could be possible to align at compile time 😏